{"name": "Проверка ручки post, что вернется ошибка, если такой пользователь уже существует", "status": "passed", "start": 1732504715998, "stop": 1732504716925, "uuid": "03aaa201-8ba4-46a8-a1b2-c938afa40c40", "historyId": "16b27191166f3b60e9cc54067a3bac35", "testCaseId": "16b27191166f3b60e9cc54067a3bac35", "fullName": "tests.test_creating_user.TestCreatingUser#test_creating_again_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creating_user"}, {"name": "subSuite", "value": "TestCreatingUser"}, {"name": "host", "value": "DESKTOP-ME9CEU9"}, {"name": "thread", "value": "2972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creating_user"}]}